imu_ekf_node:
  ros__parameters:
    use_sim_time: false
    clock_type: "ros"
    imu_topic: "/imu/data_newIMU"
    mag_topic: "/mag_newIMU"
    use_magnetometer: true
    noise_filter: false
    filter_gyro: false
    fc: 25.0
    fs: 100.0
    mag_bias_x: 0.0
    mag_bias_y: 0.0
    mag_bias_z: 0.0
    acc_noise: 0.25
    mag_noise: 0.01
    gyro_noise: 0.09
    magnetic_reference_x: 29.14
    magnetic_reference_y: -4.46
    magnetic_reference_z: 45.00
    coordinate_frame: "NED"